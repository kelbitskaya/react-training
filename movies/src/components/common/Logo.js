import React from 'react';
import { NavLink } from 'react-router-dom';

const Logo = () => (
  <NavLink to="/" className="logo">
      <strong className="logo__text">netflix</strong>
      roulette
  </NavLink>
);

export default Logo;
