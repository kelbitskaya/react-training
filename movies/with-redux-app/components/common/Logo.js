import React from 'react';

const Logo = () => (
  <a href="/" className="logo">
      <strong className="logo__text">netflix</strong>
      roulette
  </a>
);

export default Logo;
