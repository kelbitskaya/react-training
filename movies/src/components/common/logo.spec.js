import React from 'react';
import Logo from './Logo';

describe("Logo component", ()=>{
  it("should render Logo component", ()=> {
    const component = shallow(<Logo />);
    expect(component).toMatchSnapshot();
  });
});
