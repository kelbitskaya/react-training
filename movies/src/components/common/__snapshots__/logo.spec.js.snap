// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component should render Logo component 1`] = `
<NavLink
  className="logo"
  to="/"
>
  <strong
    className="logo__text"
  >
    netflix
  </strong>
  roulette
</NavLink>
`;
